base/inference.jl

```diff
@@ -574,34 +618,43 @@ function tuple_tfunc(argtype::ANY)
     argtype
 end
 
-function builtin_tfunction(f::ANY, args::ANY, argtype::ANY)
-    isva = isvatuple(argtype)
```

Comment:
i don't think `tuple` used the computation. it appears to have been used so that argument counting for intrinsic calls with a varargs tuple was handled correctly (`Core.Intrinsics.arrayref( ([1,2], 2)...)`) instead of returning Bottom